pom.xml

```diff
@@ -471,6 +471,8 @@
               <include>org.apache.hbase.thirdparty:hbase-shaded-netty</include>
               <include>org.apache.hbase.thirdparty:hbase-shaded-protobuf</include>
               <include>org.apache.htrace:htrace-core4</include>
+              <!-- afterburner module for jackson performance -->
+              <include>com.fasterxml.jackson.module:jackson-module-afterburner</include>
```

Comment:
1.  Should this also be reloacted as for others dependencies in `<relocations>` below?
2. What about other fasterxml deps?

Can you check if that all bundles have shaded `AfterburnerModule` class? For e.g. `jar tf <bundle.jar> | grep AfterburnerModule`

Reply:
yes, I have relocated this. 
Checked most of the critical bundles (spark, utilitiies, utilities-slim, hive-sync, presto and trino) and everything shows as relocated